proto/src/v1.rs

```diff
@@ -1196,6 +1197,18 @@ impl SingleStringRequest {
 }
 // Use OperationResponse here ...
 
+#[cfg(debug_assertions)]
+#[derive(Serialize, Deserialize, Debug)]
+pub struct DebugIpInfoResponse {
+    pub your_ip: IpAddr,
+}
+#[cfg(debug_assertions)]
+impl DebugIpInfoResponse {
+    pub fn new(your_ip: IpAddr) -> Self {
+        DebugIpInfoResponse { your_ip }
```

Comment:
I'd probably make this simple a Vec of IpAddr rather than a custom struct here. Saves some extra struct defs.

Reply:
I mean since it's debugging stuff I couldn't agree more

Reply:
Well also because xff is a vector of addresses :P